desktop_app/package.json

```diff
@@ -148,6 +148,7 @@
     "uuid": "^11.1.0",
     "ws": "^8.18.3",
     "zod": "^4.0.17",
+    "zod-v3": "npm:zod@^3.25.76",
```

Comment:
for the import in `desktop_app/src/backend/server/plugins/mcp/index.ts` -- I _believe_ `import { z } from "zod/v3"` _should_ work -- which if so, we won't need an extra `zod` dependency here